### PR TITLE
clarifying the APAC time

### PR DESCRIPTION
**Pull request by jimangel**
**Which issue(s) this PR fixes**:

There was a mix-up yesterday in SIG-Docs around when the APAC meeting would be held. This PR will clarify these expectations.

> Wednesday, after the fourth Tuesday, every month

Further context, SIG Docs has historically promoted our APAC meeting time as the "fourth Wednesday of every month." However, yesterday was the fourth Wednesday, but not the fourth Tuesday of January 2020. This conflicted with the US-timezone meeting (occurring weekly, except the fourth Tuesday). Lastly, the Google Calendar invite for the APAC meeting time is based off of the "fourth Tuesday" for scheduling - which further complicates things.

TL;DR timezones are hard

/cc @kubernetes/sig-docs-en-owners

### CODE REVIEW

File: sig-docs/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Covers documentation, doc processes, and doc publishing for Kubernetes.
 The [charter](charter.md) defines the scope and governance of the Docs Special Interest Group.
 
 ## Meetings
```
Comment (jaredbhatti) [inline sig-docs/README.md]: Can you move the Americas meeting time to the first slot above the APAC meeting time?

Comment (jimangel) [reply sig-docs/README.md]: Just tested... It appears that the script responsible for generating the file (`make WHAT=sig-docs generate`) will arrange descriptions alphabetically (in both `sigs.yaml` as well as the `README.md`) when executed.

We could change the descriptions to something else that places the Americas meeting first.

File: sigs.yaml [hunk 2]
```diff
@@ -1216,7 +1216,7 @@ sigs:
     day: Wednesday
     time: "02:00"
     tz: UTC
-    frequency: monthly - fourth Wednesday every month
+    frequency: monthly - Wednesday, after the fourth Tuesday, every month
```
Comment (zacharysarah) [inline sigs.yaml]:
```suggestion
    frequency: monthly - fourth Wednesday every month
```

Comment (jimangel) [reply sigs.yaml]: Chatted on slack, change is no longer recommended. Resolving.
